app/i18n/oc/user.php

```diff
@@ -30,15 +30,15 @@
 		),
 	),
 	'password' => array(
-		'invalid' => 'The password is invalid.',	// TODO - Translation
+		'invalid' => 'Lo senhal es invalid.',
```

Comment:
Even invalid is straight Occitan? Wow. :+1:

Reply:
Yes and even stronger, look:
The password is empty => Lo senhal es void
Easy to decode when you're developer, right? ^^